src/main.c

```diff
@@ -481,8 +481,14 @@ static void handle_sigint(evutil_socket_t sock, short flags, void *arg)
 		die("takeover was in progress, going down immediately");
 	if (cf_pause_mode == P_SUSPEND)
 		die("suspend was in progress, going down immediately");
-	cf_pause_mode = P_PAUSE;
-	cf_shutdown = 1;
+	if (cf_so_reuseport) {
```

Comment:
Do we want to piggy-back on SIGINT? Or should we use a new signal for this behaviour.

Reply:
Prefer new signal, see above.